### ISSUE TITLE
Shouldn't order history be inside solidus repo?

### ISSUE THREAD

**Issue by spaghetticode** [solidusio/solidus_auth_devise#123]
Hello, I am integrating Solidus ecommerce on an existing app, so I need to use my existing `User` table. 
I am not actually using this gem, but I found out that the user order history (routes, views and so on) are inside this repo.
I am a bit confused about it, because authentication does not really concern orders, IMHO these parts of code may better belong inside solidus codebase. What's your opinion on this matter?

**Comment by tvdeyen** [comment]
Yes, I agree. Would you like to propose a PR to core? This would be very much appreciated.

**Comment by spaghetticode** [comment]
Alright, I'll work on this soon. 

I also noticed that the `guest_checkout` feature is present in solidus codebase with the preference setting `allow_guest_checkout`, enabled by default, but to me it seems it's never used there. Again, its functionality is placed in the solidus_auth_devise gem.

**Comment by fthobe** [comment]
@jarednorman can be closed according to @tvdeyen in #206
